Tests/UnitTests/Mocks/MockHTTPClient.swift

```diff
@@ -31,7 +31,12 @@ class MockHTTPClient: HTTPClient {
             // swiftlint:disable:next force_try
             let data = try! JSONSerialization.data(withJSONObject: response)
 
-            self.init(response: .success(.init(statusCode: statusCode, body: data)),
+            self.init(response: .success(.init(
+                statusCode: statusCode,
+                responseHeaders: [:],
+                body: data,
+                validationResult: .notRequested
+            )),
```

Comment:
Hmm indentation here seems hard to read IMO... Maybe extract the `success` to a variable?

Reply:
Done.